redash/cli/database.py

```diff
@@ -113,10 +113,19 @@ def _reencrypt_for_table(table_name, orm_name):
         update = table_for_update.update()
         selected_items = db.session.execute(select([table_for_select]))
         for item in selected_items:
-            stmt = update.where(table_for_update.c.id == item["id"]).values(
-                encrypted_options=item["encrypted_options"]
-            )
-            db.session.execute(stmt)
+            try:
+                stmt = update.where(table_for_update.c.id == item["id"]).values(
+                    encrypted_options=item["encrypted_options"]
+                )
+            except InvalidToken:
+                logging.error(
+                    f'Invalid Decryption Key for id {item["id"]} in table {table_for_select}'
+                )
+            else:
+                db.session.execute(stmt)
+                logging.info(
+                    f'Successfully updated encryption for id {item["id"]} in table {table_for_select}'
+                )
```

Comment:
Too much redundant info?

Reply:
Good point.  We probably don't need the confirmation lines.  It's the failure ones people need to worry about. :smile: